### PR TITLE
Stub Runner initial commit.

### PR DESCRIPTION
**Pull request by PiotrZierhoffer**
What do you think? We could use that in dev process. Not pushing it tough, you decide.

### CODE REVIEW

**Comment by PiotrZierhoffer** [comment]
So what do you think?

File: Runner/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
```
Comment (konrad-kruczynski) [inline Runner/Properties/AssemblyInfo.cs:1]: Either remove it or fill the values and remove unused one. I mean the whole file.

**Comment by konrad-kruczynski** [comment]
Other than my tiny comment it's fine for the stub.

**Comment by PiotrZierhoffer** [comment]
Done, I'll merge it then.

**Pull request merged**